crepecake/src/main/java/com/google/cloud/tools/crepecake/image/LayerBuilder.java

```diff
@@ -31,17 +31,24 @@
   /**
    * The source files to build the layer from. Source files that are directories will have all
    * subfiles in the directory added (but not the directory itself).
+   *
+   * <p>The source files are specified as a list instead of a set to define the order in which they
+   * are added.
    */
-  private final Set<Path> sourceFiles;
+  private final List<Path> sourceFiles;
 
   /** The path of the file in the partial filesystem changeset. */
   private final Path extractionPath;
 
-  public LayerBuilder(Set<Path> sourceFiles, Path extractionPath) {
+  public LayerBuilder(List<Path> sourceFiles, Path extractionPath) {
     this.sourceFiles = sourceFiles;
     this.extractionPath = extractionPath;
   }
 
+  public LayerBuilder(Set<Path> sourceFiles, Path extractionPath) {
```

Comment:
Why is this one still a set?

Reply:
Changed everything to use a List instead. This way, the implementer of SourceFilesConfiguration would be given the responsibility of defining a specific and consistent ordering.